documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -37,28 +37,39 @@ When using Kafka with a Kafka cluster not managed by Strimzi, you can specify th
 [id='con-common-configuration-ssl-{context}']
 === `ssl`
 
-Use the three allowed `ssl` configuration options for client connection using a specific _cipher suite_ for a TLS version.
-A cipher suite combines algorithms for secure connection and data transfer.
+You can incorporate SSL configuration and cipher suite specifications to further secure TLS-based communication between your client application and a Kafka cluster.
+In addition to the standard TLS configuration, you can specify a supported TLS version and enable cipher suites in the configuration for the Kafka broker.
+You can also add the configuration to your clients if you wish to limit the TLS versions and cipher suites they use. 
+The configuration on the client must only use protocols and cipher suites that are enabled on the broker. 
 
-You can also configure the `ssl.endpoint.identification.algorithm` property to enable or disable hostname verification.
+A cipher suite is a set of security mechanisms for secure connection and data transfer.
+For example, the cipher suite `TLS_AES_256_GCM_SHA384` is composed of the following mechanisms, which are used in conjunction with the TLS protocol:
+
+* AES (Advanced Encryption Standard) encryption (256-bit key)
+* GCM (Galois/Counter Mode) authenticated encryption
+* SHA384 (Secure Hash Algorithm) data integrity protection
+
+The combination is encapsulated in the `TLS_AES_256_GCM_SHA384` cipher suite specification.
+
+The `ssl.enabled.protocols` property specifies the available TLS versions that can be used for secure communication between the cluster and its clients. 
+The `ssl.protocol` property sets the default TLS version for all connections, and it must be chosen from the enabled protocols.
+Use the `ssl.endpoint.identification.algorithm` property to enable or disable hostname verification. 
 
 .Example SSL configuration
 [source,yaml,subs="attributes+"]
 ----
 # ...
-spec:
+
   config:
-    ssl.cipher.suites: "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384" <1>
-    ssl.enabled.protocols: "TLSv1.2" <2>
-    ssl.protocol: "TLSv1.2" <3>
-    ssl.endpoint.identification.algorithm: HTTPS <4>
+    ssl.cipher.suites: "TLS_AES_256_GCM_SHA384" # <1>
```

Comment:
We only list a TLSv1.3 cipher so I'm still unsure if this will work in case TLSv1.2 is picked. I don't have an environment to test it but it may be simpler to add a TLSv1.2 cipher.